salt/kube-apiserver/apiserver.jinja

```diff
@@ -23,7 +24,11 @@ KUBE_ETCD_SERVERS="--etcd-cafile={{ pillar['ssl']['ca_file'] }} \
 KUBE_SERVICE_ADDRESSES="--service-cluster-ip-range={{ pillar['services_cidr'] }}"
 
 # default admission control policies
-KUBE_ADMISSION_CONTROL="--admission-control=Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,NodeRestriction,PodSecurityPolicy,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota,DefaultTolerationSeconds"
+{% if salt.caasp_pillar.get('addons:psp', True) %}
+KUBE_ADMISSION_CONTROL="--admission-control={{ (kube_admission_control + ['PodSecurityPolicy'])|join(',') }}"
+{% else %}
+KUBE_ADMISSION_CONTROL="--admission-control={{ kube_admission_control|join(',') }}"
+{% endif %}
```

Comment:
@inercia I suggested this change yesterday here for the PR against master: https://github.com/kubic-project/salt/pull/516#discussion_r186657650.

Since it got merged without the change, I think the backport to 3.0 should be the same as the master one (a clean cherry-pick). If we improve that on master we can backport the improvement too.

Reply:
Ok, I see.